litellm/litellm_core_utils/streaming_handler.py

```diff
@@ -56,14 +56,14 @@ def print_verbose(print_statement):
 
 class CustomStreamWrapper:
     def __init__(
-        self,
-        completion_stream,
-        model,
-        logging_obj: Any,
-        custom_llm_provider: Optional[str] = None,
-        stream_options=None,
```

Comment:
We cannot accept such a large change to this file. I think your formatter made this change